services/workflows-service/prisma/migrations/20230905130203_create_workflow_runtime_data_token_table/migration.sql

```diff
@@ -0,0 +1,26 @@
+-- CreateTable
+CREATE TABLE "WorkflowRuntimeDataToken" (
+    "id" VARCHAR(500) NOT NULL,
+    "token" VARCHAR(500) NOT NULL,
```

Comment:
add index over token - as it is the one that will be queried.

p.s. why is it varchar and not text as everything else?

Reply:
1. There is a unique index on it
2. Updated to text instead of varchar just for consistency. Anyway, I believe the best practice is the use varchar unless you have to use text.